boards/nordic/nrf54lv10dk/Kconfig.defconfig

```diff
@@ -1,19 +1,45 @@
 # Copyright (c) 2025 Nordic Semiconductor ASA
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 
-if BOARD_NRF54LV10DK_NRF54LV10A_CPUAPP
+# Workaround for not being able to have commas in macro arguments
+DT_CHOSEN_Z_CODE_PARTITION := zephyr,code-partition
+DT_CHOSEN_Z_SRAM_PARTITION := zephyr,sram-secure-partition
 
 config BT_CTLR
 	default BT
 
-config ROM_START_OFFSET
-	default 0 if PARTITION_MANAGER_ENABLED
-	default 0x800 if BOOTLOADER_MCUBOOT
-
 config SOC_NRF54LX_SKIP_GLITCHDETECTOR_DISABLE
 	default y
 
 config NRF_RRAM_WRITE_BUFFER_SIZE
 	default 16
 
+if BOARD_NRF54LV10DK_NRF54LV10A_CPUAPP
+
+config ROM_START_OFFSET
+	default 0 if PARTITION_MANAGER_ENABLED
+	default 0x800 if BOOTLOADER_MCUBOOT
+
 endif # BOARD_NRF54LV10DK_NRF54LV10A_CPUAPP
+
+if BOARD_NRF54LV10DK_NRF54LV10A_CPUAPP_NS
+
+config FLASH_LOAD_OFFSET
+	default $(dt_chosen_reg_addr_hex,$(DT_CHOSEN_Z_CODE_PARTITION))
+
+config FLASH_LOAD_SIZE
+	default $(dt_chosen_reg_size_hex,$(DT_CHOSEN_Z_CODE_PARTITION))
+
+# By default, if we build for a Non-Secure version of the board,
+# enable building with TF-M as the Secure Execution Environment.
+config BUILD_WITH_TFM
+	default y
+
+# By default, if we build with TF-M, instruct build system to
+# flash the combined TF-M (Secure) & Zephyr (Non Secure) image
+config TFM_FLASH_MERGED_BINARY
+	bool
+	default y
+	depends on BUILD_WITH_TFM
```

Comment:
actually why is this needed? Since I assume ncs with PM works differently to zephyr with dts